backend/route.py

```diff
@@ -7,7 +7,7 @@
 import application
 import handlers
 import auth
-import settings
+import settings as swefreq_settings
```

Comment:
YES! I've been bothered by that `import` before but didn't come up with a good fix.